package.json

```diff
@@ -179,6 +179,7 @@
     "pixrem": "5.0.0",
     "playwright-chromium": "1.28.1",
     "plop": "3.0.5",
+    "postcss": "8.4.14",
```

Comment:
I add the `postcss` to the `devDependencies` of the workspace root, as the unit test cases requires `postcss`.